### DISCUSSION TITLE
Caption or epigraph

### DISCUSSION THREAD

**Discussion by LautaroDerose** [igordanchenko/react-photo-album#177] [Ideas]
Hello, first of all thank you very much for providing this component, it has been very useful to me and I found it to be the most complete of the many I have tried.
I would like to know if it would be possible to have a description, epigraph, or caption function that is activated when the mouse hovers over a photo in the gallery.
I am using both components, reacht-photo-album and Lightbox.
I think it would be useful in photo-album to be able to show a title or description as happens with "Caption" in LightBox.
PS: Sorry for my English, I used the translator.
Greetings from Argentina!

**Comment by igordanchenko** [comment]
Hi there! This sounds like an application-specific feature and UX that can be implemented with just a few lines of code.

https://stackblitz.com/edit/react-photo-album-177?file=src%2FApp.css,src%2FApp.tsx
